cliquet/tests/resource/test_views.py

```diff
@@ -88,6 +88,24 @@ def test_permissions_are_not_modified_if_not_specified(self):
         resp = self.app.put_json(object_uri, body, headers=self.headers)
         self.assertEqual(resp.json['permissions']['read'], ['group:readers'])
 
+    def test_unknown_permissions_are_not_accepted(self):
+        self.maxDiff = None
+        body = {'data': MINIMALIST_RECORD,
+                'permissions': {'read': ['group:readers'],
+                                'unknown': ['jacques']}}
+        resp = self.app.post_json(self.collection_url, body,
+                                  headers=self.headers, status=400)
+        self.assertDictEqual(resp.json, {
+            u'errno': ERRORS.INVALID_PARAMETERS,
+            u'message': u'permissions in body: "unknown" '
+                        u'is not one of read, write',
+            u'code': 400,
+            u'error': u'Invalid parameters',
+            u'details': [{u'description': u'"unknown" is not one '
+                                          u'of read, write',
+                          u'location': u'body',
+                          u'name': u'permissions'}]})
```

Comment:
you don't need to assert the whole response format, this is not the scope of this test